src/css/inputs.css

```
@@ -114,6 +114,76 @@
   background-color: var(--color-mars-400);
 }
 
+/* File inputs */
+
+.a-input-file .a-input-file__preview-container {
```

Just a small suggestion

```suggestion
.a-input-file .a-input-file__container {
```

---

Fixed 5a51ae49c756a87276f6e8168b8d46471cf365cd